packages/foundry-sdk-generator/src/ontologyMetadata/__snapshots__/metadataResolver.test.ts.snap

```diff
@@ -270,6 +270,13 @@ exports[`Load Ontologies Metadata > Loads action types with media refs, interfac
           "pluralDisplayName": "Office",
           "primaryKey": "officeId",
           "properties": {
+            "capacity": {
+              "dataType": {
+                "type": "integer",
+              },
+              "description": "The capacity of the Office",
+              "rid": "rid",
+            },
```

Comment:
Added property so I could actually finish off the action framework with something real